iptables/iptables.go

```diff
@@ -54,11 +62,17 @@ var (
 	initOnce            sync.Once
 )
 
+// IPTable defines struct with IPVersion and few other members are added later
```

Comment:
can we remove the `and few other members are added later` for now

Reply:
done